### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by theprogrammerinyou**
Add simple navbar component to component library,

### CODE REVIEW

File: components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+.number {
+    background-color: red;
+    border-radius: 50%;
+    position: absolute;
+    top: 0;
+    right: 0;
+    color: white;
+    padding: 0.1rem;
```
Comment (Hamdrive) [inline components/badge/badge.css]: use `px` instead

File: components/card/card.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const crossImg = document.querySelector(".cross-img");
+
+const card = document.querySelector(".dismiss-card");
+const cardActions = document.querySelector(".card-actions-dismiss");
+
+const dismissCard = () => {
+    // console.log("clicked");
```
Comment (Hamdrive) [inline components/card/card.js]: cleanup comments

**Review by Hamdrive — COMMENTED** [review]
Suggest to change `px` to `rems` whereever large values are used
Also suggestion to not push so many files into one PR

**Comment by theprogrammerinyou** [comment]
The pull was taken form a single branch so all the files came along.

**Comment by theprogrammerinyou** [comment]
Done with changes, closing this PR.

**Pull request closed without merging**